tmt/templates/default/plan

```diff
@@ -0,0 +1,5 @@
+{{ plan_name }}:
```

Comment:
A general comment: can we add a suffix to all template files, `.j2`? It's a very handy mark of a file that serves as a template. And it would let us include files like `README` in this directory, if needed, without being mistaken for a template.

Reply:
Suffix added in 78d7217caecb8c3bd05d92445264de9de4bfaf02.